src/retrieve-w3c-validation/browser.ts

```diff
@@ -0,0 +1,12 @@
+// Imports
+import { W3CCSSValidatorResponse } from '..';
+
+// Utility function for retrieving response from W3C CSS Validator in a browser environment
+const retrieveForBrowser = async (url: string): Promise<W3CCSSValidatorResponse['cssvalidation']> => {
```

Comment:
Let's use the word "in" instead of "for" here. Makes more sense to me.

```suggestion
const retrieveInBrowser = async (url: string): Promise<W3CCSSValidatorResponse['cssvalidation']> => {
```